static/compiled/application.css

```diff
@@ -0,0 +1,1171 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
@pwjablonski make sure you’ve merged/rebased in the latest master, then do `git rm -r static`